config/config/src/main/java/io/helidon/config/internal/FileOverrideSource.java

```diff
@@ -63,13 +63,29 @@ protected String uid() {
         Optional<byte[]> digest = dataStamp();
         LOGGER.log(Level.FINE, String.format("Getting content from '%s'.", filePath));
 
-        try {
-            OverrideData overrideData = OverrideSource.OverrideData
-                    .create(new StringReader(FileSourceHelper.safeReadContent(filePath)));
-            return new Data<>(Optional.of(overrideData), digest);
-        } catch (IOException e) {
-            throw new ConfigException("Cannot load data from source.", e);
-        }
+        OverrideData overrideData = OverrideSource.OverrideData
+                .create(new StringReader(FileSourceHelper.safeReadContent(filePath)));
+        return new Data<>(Optional.of(overrideData), digest);
+
+    }
+
+    /**
+     * Create a new file override source from meta configuration.
+     *
+     * @param metaConfig meta configuration containing the {@code path} and other configuration options
+     * @return a new file override source
+     */
+    public static FileOverrideSource create(Config metaConfig) {
+        return builder().config(metaConfig).build();
+    }
+
+    /**
+     * Create a new fluent API builder.
```

Comment:
Cut and paste error?  What API are we building?

Reply:
Not sure. I consider "fluent API" to be the name of the design pattern using Builders and in general: classes, that return the same instance when something is updated.